v8/pretty_breadcrumbs/README.md

```diff
@@ -0,0 +1,16 @@
+
+This plugin converts filepaths from breadcrumbs into more pretty
+breadcrumbs based on metadata information defined in the source
+file.
+
+For example, if file `test/index.rst` contains
+
+```
+.. crumb: This is a test
+```
+
+the breadcrumb generated would be `This is a test` instead of `test`.
```

Comment:
how about defaulting to titles? Try `crumb` first, if it doesn’t exist, use `title`

Reply:
Titles are usually quite long and I think the filepath (slug) is better than the title if no `crumb` is defined.

But maybe this should be a config option.